synapse/storage/databases/main/event_federation.py

```diff
@@ -903,8 +916,8 @@ def get_insertion_event_backward_extremities_in_room_txn(
                 (
                     room_id,
                     self._clock.time_msec(),
+                    BACKFILL_EVENT_EXPONENTIAL_BACKOFF_MAXIMUM_DOUBLING_STEPS,
                     1000 * BACKFILL_EVENT_EXPONENTIAL_BACKOFF_STEP_SECONDS,
```

Comment:
Probably be clearer to make the constant milliseconds?

Reply:
c713c8ec7